cmd/network/verification.go

```diff
@@ -131,6 +131,7 @@ func NewCmdValidateEgress() *cobra.Command {
 type egressVerificationAWSClient interface {
 	DescribeSubnets(ctx context.Context, params *ec2.DescribeSubnetsInput, optFns ...func(options *ec2.Options)) (*ec2.DescribeSubnetsOutput, error)
 	DescribeSecurityGroups(ctx context.Context, params *ec2.DescribeSecurityGroupsInput, optFns ...func(options *ec2.Options)) (*ec2.DescribeSecurityGroupsOutput, error)
```

Comment:
From the looks of this code, I think we can blow away this function from the interface.

Reply:
Thanks, I will remove it!

Reply:
Looks like I missed that somehow - disregard this.